ext/node/Cargo.toml

```diff
@@ -14,6 +14,8 @@ description = "Node compatibility for Deno"
 path = "lib.rs"
 
 [dependencies]
+aes = "0.8.2"
+cbc = "0.1.2"
```

Comment:
Consider moving these two to root `Cargo.toml` and using `aes.workspace = true` here (make sure to update other references in various `Cargo.toml` files)